tests/environment-setup/test_used_docker_compose.py

```diff
@@ -0,0 +1,61 @@
+# pylint: disable=unused-argument
+# pylint: disable=bare-except
+# pylint: disable=redefined-outer-name
+
+import re
+from pathlib import Path
+from typing import List, Tuple
+
+import pytest
+
+
+@pytest.fixture(scope="module")
```

Comment:
why not session scoped?

Reply:
well, at the end of the day, this is only going to be used in test_used_docker_compose module. I think in this case, effectively it is the same session as module, right?